### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Shobelish**
Please review changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # PracticeScripts
 All other sample scripts
 Sample
+Readme provides information on what to expect in Software and how to get started
```
Comment (shobanasat) [inline README.md:4]: Add some more details

**Review by shobanasat — CHANGES_REQUESTED** [review]
Add some more details

File: README.md (hunk 1 above)
Comment (Shobelish) [reply README.md:4]: Added details as requested

**Review by shobanasat — APPROVED** [review]

**Comment by shobanasat** [comment]
All fine now

**Review by shobanasat — APPROVED** [review]

**Comment by shobanasat** [comment]
Merged to master

**Pull request closed without merging**